sbt/src/main/scala/laika/sbt/Tasks.scala

```diff
@@ -232,7 +232,7 @@ object Tasks {
       System.in.read
     }
     finally {
-      streams.value.log.info(s"Shutting down preview server.")
+      streams.value.log.info(s"Shutting down preview server...")
```

Comment:
These dots are ominous 😂

Reply:
Just supporting the most impatient souls amongst us, so that they don't think something is broken while the 2 seconds pass... 🙂